### ISSUE TITLE
states-of-matter depends on atomic-interactions

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/states-of-matter#48]
(1) The dependency on atomic-interactions is missing from package.json.

(2) This is the exact same problem that we have with fluid-pressure-and-flow and under-pressure, see https://github.com/phetsims/under-pressure/issues/118. atomic-interactions is a single screen from states-of-matter. We agreed that this should be fixed fluid-pressure-and-flow and under-pressure, why shouldn't it be fixed here too?

**Comment by pixelzoom** [comment]
Places where states-of-matter is using atomic-interactions shown in screenshot below. This is identical to how FPAF is using under-pressure.

![screenshot_223](https://cloud.githubusercontent.com/assets/3046552/6405925/c6369d4e-bde6-11e4-927c-9edc205bfa80.png)

**Comment by jbphet** [comment]
Agreed - this should be reorganized.  I don't know that we ever communicated the requirement that Atomic Interactions should reach into States of Matter, and not the other way around, to ActualConcepts.  I will take on the reorganization and will communicate this policy to ActualConcepts so that they are aware of it for future sim development.

**Comment by jbphet** [comment]
I've moved the code from the atomic-interactions repository into the states-of-matter repository, so the dependency issue has been addressed.  I tried to preserve the history of the atomic-interaction files by following the process described here: http://gbayer.com/development/moving-files-from-one-git-repository-to-another-preserving-history/.  This doesn't appear to have worked, since when I look at the file's history on GitHub, all I see is the commit message about moving the file from the other repo.  So, if anyone ever needs to find history on one of the atomic-interactions files, they will need to go look in that repo.

At any rate, the main issue is addressed, so this will be closed.
